apps/shelter-web/src/app/pages/home/home.tsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from 'react-router-dom';
+
+import { gql, useQuery } from '@apollo/client';
+
+export const GET_CURRENT_USER = gql`
+  query GetCurrentUser {
+    currentUser {
+      email
+      username
+    }
+  }
+`;
+
```

Comment:
Let's make a follow up ticket set up Apollo codegen to generate hooks for this project.

Reply:
@vecchp  - FYI created [DEV-1111](https://betterangels.atlassian.net/browse/DEV-1111) with parent [SDB-21](https://betterangels.atlassian.net/browse/SDB-21)